sky/skylet/providers/oci/config.py

```diff
@@ -0,0 +1,87 @@
+"""
+OCI Configuration.
+History:
+ - Hysun He (hysun.he@oracle.com) @ Apr, 2023: Initial implementation
+ 
+"""
+import logging
+import os
+from sky import skypilot_config
+from typing import Optional
+
+logger = logging.getLogger(__name__)
+
+
+class oci_conf:
+    IMAGE_TAG_SPERATOR = "|"
+    INSTANCE_TYPE_RES_SPERATOR = "$_"
+    CPU_MEM_SPERATOR = "_"
+
+    DEFAULT_NUM_VCPUS = 8
+    DEFAULT_MEMORY_CPU_RATIO = 6
```

Comment:
After checking the catalog, it seems we do have the `4x` version for the CPU instance. Let's use `4` here to make it align with the other cloud. I think it should be fine to be different from the default value of the one in the console, but it would be better to keep it similar to the other clouds. With that, the user can have a fair comparison of the prices among the clouds, and I believe OCI offers pretty competitive price. : )

Reply:
Sounds good. thanks.  Will change the MEMORY_CPU_RATIO to 4. And the catalog file submitted yesterday included the 4X shape already.